cmd/importcmd/imagebundle/image_bundle.go

```diff
@@ -43,22 +44,42 @@ func NewCommand(out output.Output) *cobra.Command {
 			cleaner.AddCleanupFn(func() { _ = os.RemoveAll(tempDir) })
 			out.EndOperation(true)
 
-			out.StartOperation("Unarchiving image bundle")
-			err = archive.UnarchiveToDirectory(imageBundleFile, tempDir)
-			if err != nil {
-				out.EndOperation(false)
-				return fmt.Errorf("failed to unarchive image bundle: %w", err)
-			}
-			out.EndOperation(true)
+			sort.Strings(imageBundleFiles)
 
-			out.StartOperation("Parsing image bundle config")
-			cfg, err := config.ParseImagesConfigFile(filepath.Join(tempDir, "images.yaml"))
-			if err != nil {
-				out.EndOperation(false)
-				return err
+			var cfg config.ImagesConfig
+
+			// Just in case users specify the same bundle twice, keep a track of
+			// files that have been extracted already so we only extract each of them once.
+			extractedBundles := make(map[string]struct{}, len(imageBundleFiles))
```

Comment:
[nit] I think with type bool, intend would be easier to understand.

```suggestion
			extractedBundles := make(map[string]bool, len(imageBundleFiles))
```

Reply:
@jimmidyson already used this format in a different PR, I would prefer consistency.

Reply:
It's pretty common practice to use `struct{}` as marker in map.